pkg/api/norman/customization/project/project_actions.go

```diff
@@ -459,23 +476,29 @@ func isProvisionedRke2Cluster(cluster *v3.Cluster) bool {
 	return cluster.Status.Provider == v32.ClusterDriverRke2 && imported.IsAdministratedByProvisioningCluster(cluster)
 }
 
-func areK3SPodSecurityPoliciesEnabled(managementCluster *v3.Cluster, provisioningCluster *provisioningv1.Cluster) bool {
-	if managementCluster.Status.Provider != v32.ClusterDriverK3s {
-		return false
-	}
-	args := parseKubeAPIServerArgs(provisioningCluster)
-	return strings.Contains(args["enable-admission-plugins"], "PodSecurityPolicy")
-}
-
 // parseKubeApiServerArgs parses the "kube-apiserver-arg" available in the
 // clusters' MachineGlobalConfig to a map. The arguments are expected to
 // follow the "key=value" format. Arguments that don't follow this format
 // are ignored.
 func parseKubeAPIServerArgs(provisioningCluster *provisioningv1.Cluster) map[string]string {
-	result := map[string]string{}
-	args := provisioningCluster.Spec.RKEConfig.MachineGlobalConfig.Data["kube-apiserver-arg"].([]any)
+	result := make(map[string]string)
+
+	rawArgs, ok := provisioningCluster.Spec.RKEConfig.MachineGlobalConfig.Data["kube-apiserver-arg"]
+	if !ok || rawArgs == nil {
+		return nil
+	}
+
+	args, ok := rawArgs.([]any)
+	if !ok || args == nil {
+		return nil
+	}
+
 	for _, arg := range args {
-		key, value, found := strings.Cut(arg.(string), "=")
+		s, ok := arg.(string)
+		if !ok {
```

Comment:
Is this possible? If not, should we log an error here?

Reply:
@jakefhyde He initially had a log statement here which I pointed out. The `project` package doesn't seem to have any logging, so I wasn't sure if we wanted to add in a log for this scenario alone, or if a logger needed any prerequisite setup. 

If we do want to add logging for this, then my bad Simon!

Reply:
You bring up a good point, if there's no logging in this package yet we are probably fine to just omit it.